File: domain/src/test/java/no/nav/sbl/dialogarena/sendsoknad/domain/util/KommuneTilNavEnhetMapperTest.java
package no.nav.sbl.dialogarena.sendsoknad.domain.util;

import org.junit.Test;

import static no.nav.sbl.dialogarena.sendsoknad.domain.util.KommuneTilNavEnhetMapper.getOrganisasjonsnummer;
import static org.junit.Assert.*;

public class KommuneTilNavEnhetMapperTest {

    @Test
    public void TestGetOrganisasjonsnummerOfNull() {
        String result = getOrganisasjonsnummer(null);
        assertNull(result);
    }
}

File: src/test/kotlin/no/nav/sosialhjelp/soknad/common/mapper/KommuneTilNavEnhetMapperTest.kt
package no.nav.sosialhjelp.soknad.common.mapper

import org.assertj.core.api.Assertions.assertThat
import org.junit.jupiter.api.Test

internal class KommuneTilNavEnhetMapperTest {

    @Test
    fun testGetOrganisasjonsnummerOfNull() {
        val result = KommuneTilNavEnhetMapper.getOrganisasjonsnummer(null)
        assertThat(result).isNull()
    }
}
